Styles/Components/alert.css

```diff
@@ -0,0 +1,101 @@
+.alert {
+  border: transparent 1px solid;
+  border-radius: 8px;
+  width: 70%;
+  padding: 0.75rem;
+  font-size: 1.2rem;
+}
+
+.alert-success {
+  background: var(--green-2);
+  color: var(--black-1);
+}
+
+.alert-warning {
+  background: var(--orange-2);
+  color: var(--black-1);
+}
+
+.alert-error {
+  background: var(--red-2);
+  color: var(--black-1);
+}
+
+.alert-info {
+  background: var(--blue-2);
+  color: var(--black-1);
+}
+
+.alert-icon {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+}
+
+.alert-icon .icon {
+  width: 1.5rem;
+  height: 1.5rem;
+  margin-right: 0.5rem;
```

Comment:
Use global variables for spacing properties like margin, padding, gap to maintain consistency across the web app